site/templates/film/item.jet

```diff
@@ -62,94 +80,106 @@
           {{end}}
 
           {{yield awardNominations() film}}
-          <s72-element-switcher>
-            {{if isEnabled("element_switcher_enabled")}}
-              <div class="meta-detail-switcher-tagline">
-                <p>{{film.Tagline}}</p>
-              </div>
-            {{end}}
-            <div class="{{ isEnabled("element_switcher_enabled") ? "s72-hide" : "" }}"> {* hide class prevents FOUC *}
-              <div class="meta-detail-synopsis">{{film.Overview | raw}}</div>
-              <div class="meta-detail-cast">
-              {{if len(film.Cast) > 0 }}
-                <h2>{{i18n("meta_detail_cast_title")}}</h2>
+          <div class="element-switcher-sponsor">
+            <s72-element-switcher class="meta-detail-info-width">
```

Comment:
I'm generally of the opinion that we shouldn't put classes (particularly ones with structural stuff) directly on Relish components.  If a component winds up having no child elements for whatever reason, it should have no impact on the rest of the DOM.  In this case, it looks like the class on the element switcher is being used to push the sponsor image to the right... I feel the sponsor image itself or the wrapper should be responsible for that behaviour, not the element-switcher.

Reply:
Yeah that makes sense to me, agreed.

That class isn't being used to push the logo to the right necessarily (although it is doing that), instead it has three css properties that:
- Applies md breakpoint 563px width that was previously on `.meta-detail-content`
- Applies xl breakpoint 1100px width that was previously on `.meta-detail-content`
- Applies `flex: 1 1 100%;` so it behaves as normal unless the logo is beside it.

Note have removed `.meta-detail-info-width` for a [clearer css strategy](https://github.com/shift72/core-template/pull/320/files#diff-b9add1ee54c4a860fd60c71e1c243453ea9f48c98a3dd2596777b2239f6436cdR154-R165)